site/content/blog/2022-08-22-the-param-miner-addon/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "The Param Miner Add-on"
+summary: >
+  The parameter discovery add-on for ZAP.
+images:
+- https://www.zaproxy.org/blog/2022-08-22-the-param-miner-addon/images/paramminer.png
+type: post
+tags:
+- blog
+date: "2022-08-22"
+authors: 
+    - Arkaprabha
+---
+
+Parameter discovery is the process of finding unlinked, unreferenced, or hidden parameters that can be used by the user to control or influence the behaviour of the application. Resulting in successful attack scenarios like SQL injection, XSS, SSRF, Web Cache Poisoning based attacks and much more. Often times such uncovered parameters are properly configured and can be a suitable entry point for further manipulation. These are real hidden treasures which can be very crucial in the recon process to develop a successful exploit.  
```

Comment:
`SQL Injection`, `XSS` etc could all link to the relevant ZAP issues? eg https://www.zaproxy.org/docs/alerts/40018/ ...